tests/gradients/core/test_hadamard_gradient.py

```diff
@@ -601,30 +604,36 @@ def test_requested_wire_not_exist(self, device_wires):
             qml.expval(qml.PauliZ(0) @ qml.PauliX(1))
 
         tape = qml.tape.QuantumScript.from_queue(q)
-        with pytest.raises(
-            qml.QuantumFunctionError,
-            match="The requested auxiliary wire does not exist on the used device.",
-        ):
+        _match = "The requested auxiliary wire does not exist on the used device"
+        with pytest.raises(qml.wires.WireError, match=_match):
             qml.gradients.hadamard_grad(tape, aux_wire=aux_wire, device_wires=dev.wires)
 
-    @pytest.mark.parametrize("aux_wire", working_wires + already_used_wires)
```

Comment:
Were the `working_wires` already not used in the test before? (seems so from the diff)

Reply:
This is a list addition, so they were being used. However, this PR changes which error is being raised for an auxiliary wire that is being requested but already in use by the original circuit. Therefore the cases I effectively removed here are instead  tested in `test_device_wire_does_not_exist` below :)

Reply:
Right, thanks!